### ISSUE TITLE
Item popup title missing ARIA heading role

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-hotgraphic#265]
### Subject of the issue
The popup item title is read by a screen reader but it isn't announced as a 'heading' neither is the heading `aria-level` announced.

### Your environment
* 5.28.15 FW / 6.6.4 Hotgraphic
* Mac Safari running voiceOver

### Steps to reproduce
With a screen reader running, select a Hotgraphic item to trigger the popup. Note the item title is readable but it isn't announced as a 'heading' neither is the heading `aria-level`.
